containers/daap-python-base/src/var/task/requirements.txt

```diff
@@ -0,0 +1 @@
+boto3==1.28.9
```

Comment:
Current version is `1.28.20` but as I understand, `boto3` needs to be pinned at a specific version due to a downstream package, `mojap-metadata` in `athena-load` image?